README.md

```diff
@@ -121,24 +121,20 @@ Tarball (choose the appropriate version and platform):
 ```
 $ curl -sfLO https://github.com/intelsdi-x/snap/releases/download/0.18.0/snap-0.18.0-linux-amd64.tar.gz
 $ tar xf snap-0.18.0-linux-amd64.tar.gz
-$ cp snapd /usr/local/bin
-$ cp snapctl /usr/local/bin
+$ cp snapteld /usr/local/sbin
+$ cp snaptel /usr/local/bin
 ```
 
-Ubuntu 16.04.1 [snapd package version 2.13+](https://launchpad.net/ubuntu/+source/snapd) installs snapd/snapctl binary in /usr/bin. These executables are not related to snap-telemetry. Running `snapctl` from snapd package will result in the following error message:
+NOTE: snap-telemetry packages prior to 0.19.0 installed `/usr/local/bin/{snapctl|snapd}` and these binaries have been renamed to `snaptel` and `snapteld`. snap-telemetry packages prior to 0.18.0 symlinked `/usr/bin/{snapctl|snapd}` to `/opt/snap/bin/{snapctl|snapd}` and may cause conflicts with [Ubuntu's `snapteld` package](http://packages.ubuntu.com/xenial-updates/snapd). Ubuntu 16.04.1 [snapteld package version 2.13+](https://launchpad.net/ubuntu/+source/snapd) installs snapd/snapctl binary in /usr/bin. These executables are not related to snap-telemetry. Running `snapctl` from snapd package will result in the following error message:
```

Comment:
`and may cause conflicts with [Ubuntu's >>>>>`snapteld`<<< package]`

`>>>snapteld<<< package version 2.13+`